CHANGELOG.md

```diff
@@ -12,6 +12,58 @@ rely on parsing the output from multiple nodes should pass the `--log-dir` flag
 to `tsh ssh`, which will create a directory where the separated output of each node
 will be written.
 
+#### Container Images
+
+Teleport 15 contains several breaking changes to improve the default
+security and usability of container images.
+
+##### "Fat" container images are discontinued
+
+In order to increase default security in 15+, Teleport will no longer publish
+[container images containing a shell and rich command line environment](https://github.com/gravitational/teleport/blob/branch/v14/build.assets/charts/Dockerfile)
+to https://gallery.ecr.aws/gravitational/teleport. Instead, all users should
+use the [distroless images](https://github.com/gravitational/teleport/blob/branch/v15/build.assets/charts/Dockerfile-distroless)
+introduced in Teleport 12. These images can be found at:
+
+* https://gallery.ecr.aws/gravitational/teleport-distroless
+* https://gallery.ecr.aws/gravitational/teleport-ent-distroless
+
+For users who need a shell in a Teleport container, a "debug" image is
+available which contains BusyBox, including a shell and many CLI tools. Find the debug images at:
+
+* https://gallery.ecr.aws/gravitational/teleport-distroless-debug
+* https://gallery.ecr.aws/gravitational/teleport-ent-distroless-debug
+
+Do not run debug container images in production environments.
+
+Fat container images will continue to be published for Telport 13 and 14
```

Comment:
s/Fat/Heavy/

Reply:
fixed in https://github.com/gravitational/teleport/pull/35452/commits/8407fc98aa1005736ecef01fe60d1e47cf140dea